src/main/java/org/fundacionjala/coding/nestor/Rental.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.nestor;
+
+public class Rental {
+    private Movie _movie;
+    private int _daysRented;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_daysRented" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)